src/utils/additional-headers.ts

```diff
@@ -1,9 +1,11 @@
 export const getAdditionalHeaders = (
-  serviceTitle: string,
+  serviceName?: string,
   sourceName?: string,
   sourceVersion?: string,
 ) => {
-  const requestInformation = `${serviceTitle} | ${sourceName ? `${sourceName} using ` : ''}JS SDK`;
+  const serviceNamePart = serviceName ? `${serviceName} | ` : '';
+  const sourceNamePart = sourceName ? `${sourceName} using ` : '';
+  const requestInformation = `${serviceNamePart}${sourceNamePart}JS SDK`;
```

Comment:
It will makes more sense if the first part would be based on sourceName and second part on service name (and view might be better than service)